src/settings.rs

```diff
@@ -174,6 +177,10 @@ fn default_connection_pool_size() -> usize {
     DEFAULT_POOL_SIZE
 }
 
+fn default_message_timeout_tics() -> u64 {
+    10
```

Comment:
is it now 10 periods instead of 2? Why 10?

Reply:
See PR description. IMO, seems reasonable to raise it to 10, as reelection only starts at 20 ticks.